airflow/listeners/listener.py

```diff
@@ -33,6 +34,8 @@
 
 _listener_manager = None
 
+_scheduler_hooks = ["on_dag_run_success", "on_dag_run_failure"]
```

Comment:
Given these hooks are also called from backfill:

```suggestion
_dagrun_hooks = ["on_dag_run_success", "on_dag_run_failure"]
```

Reply:
Removed those